<launch>
	<arg name="namespace" default="/"/>
	<node pkg="dji_sdk_read_cam" type="dji_sdk_read_cam" name="dji_sdk_read_cam" ns="$(arg namespace)">
		<!-- node parameters -->
		<param name="output_image_color" type="bool" value="true"/>
		<param name="image_width" type="int" value="1280"/>
		<param name="image_height" type="int" value="720"/>
		<param name="output_image_width" type="int" value="640"/>
		<param name="output_image_height" type="int" value="480"/>
	</node>
</launch>
